package.json

```diff
@@ -33,13 +33,13 @@
   },
   "dependencies": {
     "@types/seedrandom": "^3.0.1",
-    "apexcharts": "^3.29.0",
+    "apexcharts": "^3.31.0",
```

Comment:
Gibt es dafür einen geschickten Befehl, um alles mal auf den neuesten Stand zu bringen? Würde ich in Zukunft lieber in nem separaten Commit machen.

Reply:
Ja, das geht mit `npm update`. Und ja, wäre in einem separaten PR eigentlich besser gewesen. :-(